sys/benchmark/benchmark.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     sys_benchmark
+ * @{
+ *
+ * @file
+ * @brief       Utility functions for the benchmark module
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
```

Comment:
here and below: usually we put a new line after system headers

Reply:
I usually do this, too, but in cases where there is only a small (roughly up to 2 or 3 ) includes in a file I tend to put them in a single block, as I think its quite ugly to have a single include per 'newline separated block'... But will fix this here to make you happy :-)